src/backend/services/wallet-api/endpoints.md

```diff
@@ -1,65 +1,68 @@
 # Wallet API
 
-## Create a new wallet for a user
+## Get the user's wallet amount
 
-> `POST` <http://localhost/wallet/create>
+> `POST` <http://localhost/wallet/amount>
 
 **Required JSON:**
 
-```json
+```json 
 {
   "umnetId": string
 }
 ```
-
-Registers a wallet for a new user in the blockchain and returns the server message.
+Returns the amount of bisoncoin that the user currently owns.
 
 **Example return:**
 
 ```json
-201
 {
-  "success": boolean
+  "amount": int
 }
 ```
 
 or
 
 ```json
-400
 {
-  "error"  : string
+  "error" : string
 }
 ```
 
-## Get the user's wallet amount
+## Get the user's transaction history
 
-> `POST` <http://localhost/wallet/amount>
-
-**Required JSON:**
+> `GET` <http://localhost/wallet/history?walletId={SearchID}>
 
```

Comment:
change this file when changing the route

Reply:
done